examples/user_interfaces/embedding_in_gtk_sgskip.py

```diff
@@ -25,8 +25,8 @@
 
 f = Figure(figsize=(5, 4), dpi=100)
 a = f.add_subplot(111)
-t = arange(0.0, 3.0, 0.01)
-s = sin(2*pi*t)
+t = np.arange(0.0, 3.0, 0.01)
+s = np.sin(2*np.pi*t)
```

Comment:
Do we have a convention, if this should include spaces around the operators? PEP8 would want them. Throughout the examples both variants are used.

Reply:
See discussion at https://github.com/matplotlib/matplotlib/pull/7562#issuecomment-264775106 which I now agree with: I still believe spaces are in general good, but in some cases no spaces read better.
See also https://www.python.org/dev/peps/pep-0008/#other-recommendations ("use your own judgment", hehe).